shell/packages/sandstorm-accounts-ui/accounts-ui-server.js

```diff
@@ -25,8 +25,13 @@ Meteor.users.deny({
 Meteor.publish("getMyUsage", function () {
   if (this.userId) {
     // TODO(someday): Make this reactive.
-    var usage = this.connection.sandstormDb.getMyUsage(Meteor.users.findOne(this.userId));
-    this.added("users", this.userId, {pseudoUsage: usage});
+    var user = Meteor.users.findOne(this.userId);
+    var usage = this.connection.sandstormDb.getMyUsage(user);
+    var referralBonus = this.connection.sandstormDb.getMyReferralBonus(user);
```

Comment:
Small sylistic thing, consider doing `var db = this.connection.sandstormDb` to save on repetition. I've started doing that at the top of all my meteor methods, even if I only use the db once.